modules/reports/server-ts/index.ts

```diff
@@ -0,0 +1,15 @@
+import ServerModule from '@module/module-server-ts';
+
+import Report from './sql';
+import schema from './schema.graphql';
+import createResolvers from './resolvers';
+import pdfResolvers from './generateFile/pdf/resolvers';
```

Comment:
`pdf` and `excel` should come as separate modules. Also, they shouldn't be located in the `generateFile` folder, it is redundant.